hound/config.json

```diff
@@ -0,0 +1,190 @@
+{
+	"max-concurrent-indexers": 2,
+	"dbpath": "data",
+	"repos": {
+		"kubernetes": {
+			"url": "https://github.com/kubernetes/kubernetes.git",
```

Comment:
no way to say "all repos under an org" ?  feature request time?

Reply:
will check if they support this.